article/app/views/fragments/emailArticleBody.scala.html

```diff
@@ -81,9 +81,15 @@ <h2 class="block__title">@title</h2>
                     }
 
                     case ImageBlockElement(media, data, showCredit) => {
-                        @EmailImage.bestFor(media).map { url =>
+                        @EmailImage.bestFor(media).map { imageUrl =>
                             @fullRow {
-                                <img width="580" class="full-width" src="@url" @data.get("alt").map { alt => alt="@alt" }>
+                                @block.url.filter(_ => article.isLiveBlog).fold {
+                                    <img width="580" class="full-width" src="@imageUrl" @data.get("alt").map { alt => alt="@alt" }>
```

Comment:
would it make sense to have an helper function in EmailHelpers for this image Html? It seems to be used several times in this template.